ideascube/views.py

```diff
@@ -25,23 +25,49 @@
 from ideascube.library.models import Book
 from ideascube.mediacenter.models import Document
 
+# Do not import directly Catalog as we want to mock it for test.
+from ideascube.serveradmin import catalog as catalog_mod
```

Comment:
As said in the comment, it is to be able to mock the Catalog class.

If we `from ideascube.serveradmin.catalog import Catalog`, the Catalog class is directly referenced. So a mock.patch of `ideascube.serveradmin.catalog.Catalog` will have no effect.